plan/planbuilder.go

```diff
@@ -1058,12 +1058,11 @@ func (b *planBuilder) buildExplain(explain *ast.ExplainStmt) Plan {
 	p := &Explain{StmtPlan: pp}
 	switch strings.ToLower(explain.Format) {
 	case ast.ExplainFormatROW:
-		retFields := []string{"id", "parents", "children", "task", "operator info"}
+		retFields := []string{"id", "parents", "children", "task", "operator info", "count"}
```

Comment:
Why this is changed?

Reply:
I think it's ok to make the column type for `count` to be string, so we can store all the pre-generated explain information to a `[][]string`, not a `[][]Datum`